lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -94,4 +94,22 @@ describe('NxThreatIndicator', function() {
     expect(icon).toHaveClassName('nx-threat-indicator');
     expect(icon).toHaveClassName('nx-threat-indicator--low');
   });
+
+  it('should hide tooltip when presentational prop is set true', function() {
+    const tooltip = getShallowComponent({ presentational: true }).find(NxToolip);
+
+    expect(tooltip).not.toExist();
```

Comment:
Might want to assert that the icon is still there

Reply:
> Might want to assert that the icon is still there

Added it here: 94dceaf99a0ca4ae5d4afb8bc05b2ab3d8553bae